### PR TITLE
Add the 605 global theme

### PR DESCRIPTION
**Pull request by dimkr**
@mrfricks Do you have the original wallpaper from 6.0.5 in SVG format?

### CODE REVIEW

**Comment by peabee** [comment]
I don't know what this theme looks like (or why it is needed) - what does "605" refer to? TahrPup?
Shouldn't it be a new separate rootfs-package so that it can be optional in builds? i.e. jwm-config and ptheme should not be "extended".

**Comment by dimkr** [comment]
> what does "605" refer to? TahrPup?

Yes

**Review by 01micko — APPROVED** [review]

File: woof-code/rootfs-skeleton/usr/share/backgrounds/605.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1920" height="1080" viewBox="0 0 1066.666 600"><path style="fill:#dee6ff;fill-opacity:1" d="M0 0h1920v1080H0z"/><g transform="matrix(12 0 0 12 -11354.526 -4740.696)"><path d="M999.555 412.889c3 0 6 2.7 6 6.3-.03 3.3-2.4 6.6-6 6.6s-6.6-3-6.6-6.6h1.8c0 1.2 1.2 2.7 3 2.7s2.7-1.5 2.7-3c0-1.8-1.5-3-2.7-3-1.8 0-3 1.5-3 3v.3h-1.8c0-3.6 2.4-6.3 6.6-6.3z" style="fill:#c5d8e2;fill-opacity:1;stroke:none"/><circle cx="979.055" cy="413.889" r="3.3" style="fill:#c5d8e2;fill-opacity:1;stroke:none"/><path d="m990.555 421.389-7.8-2.4c-.9-.3-1.8-.3-1.2 1.2l2.7 4.8c.3 1.2 1.3.2 1.2.3l5.1-2.7c1.2-.6 0-1.2 0-1.2z" style="fill:#c5d8e2;fill-opacity:1;stroke:none"/><path style="fill:none;stroke:#c5d8e2;stroke-width:.9;stroke-linecap:round;stroke-opacity:1" d="M992.355 427.389c-.6 1.8-4.8 2.4-6.6.6-1.5-1.5-1.5-3-1.5-3s-1.2 1.8-3.6 1.8c-3.9 0-3.9-4.2-3.9-4.2"/></g></svg>
```
Comment (dimkr) [inline woof-code/rootfs-skeleton/usr/share/backgrounds/605.svg]: Contrast problem here, the desktop icon labels are white and the background is super bright

**Comment by dimkr** [comment]
> Shouldn't it be a new separate rootfs-package so that it can be optional in builds

Without this PR, woof-CE has only 3 global themes that cover both GTK+ 2 and 3 with nearly identical appearance, and it's nice to have a fourth one. GTK+ 2 purists that still ignore GTK+ 3 in 2022 can use the 605 theme without the gtk_theme_stark_blueish petbuild, but it won't support GTK+ 3 (as today).

(Pretty much any >= 2018 Puppy, including bionicpup32-8.0+29-uefi.iso, includes some version of stark-blueish and Puppy Standard.)

**Review by 01micko — APPROVED** [review]

**Comment by mrfricks** [comment]
@dimkr sorry, only just seen this.  No i haven't got an .svg of the wallpaper but i did find this link to the transparent .png that was probably used to create the original. 
https://oldforum.puppylinux.com/download/file.php?id=4905
It's odd as i'm pretty sure that the wallpaper was part of woofce when building tahrpup but a search doesn't find it

Vincent_1_and_2

**Comment by dimkr** [comment]
Thanks @mrfricks!

> It's odd as i'm pretty sure that the wallpaper was part of woofce when building tahrpup but a search doesn't find it

That's interesting, I'll see if I can grep woof-CE's history for addition or deletion of images.

@01micko Do you have a recommendation for some tool that can trace a .png back to SVG?

**Comment by 01micko** [comment]
Nope. I usually do it in mtpaint manually, but perhaps some tool exists.

**Comment by 01micko** [comment]
OMG.. I think `mkwallpaper` can do it - all vector, no base64 encoding (cairo based).
Gimme some time, pretty sure it's doable. (super busy in real life ATM).

**Comment by mrfricks** [comment]
here's one from inkscape but it's missing the shading on the bellies and legs .... approaching 4k lol
vincents

**Comment by 01micko** [comment]
Nope - cairio _is_ doing base64 :crying_cat_face: 

foo

**Comment by dimkr** [comment]
lol, I'll see if I can do it with `potrace`

**Comment by mrfricks** [comment]
getting there
vincents 2

**Comment by mrfricks** [comment]
vincents 3

**Comment by mrfricks** [comment]
the background colour isn't exact but can be easily changed if you load it up into inkscape
original background colour is #DDE5F (dde5feff in inkscape)

colour1

**Comment by dimkr** [comment]
36K, scaled to 1080p so it doesn't render to slowly, and I used a slightly darker background so the white desktop icon labels are readable. Let's see if it's dark enough.

**Pull request merged**